raster/src/main/scala/geotrellis/raster/ConstantTile.scala

```diff
@@ -150,7 +150,7 @@ trait ConstantTile extends Tile {
     * @param   f  A function from Int to Int, executed at each point of the tile
     * @return     The result, a [[Tile]]
     */
-  def map(f: Int => Int): Tile = IntConstantTile(f(iVal), cols, rows)
+  def map(f: Int => Int): Tile
```

Comment:
same problem exists in `def mapDouble` and should be addressed to keep the behavior consistent.